types/react-native/test/index.tsx

```diff
@@ -368,10 +368,15 @@ export class FlatListTest extends React.Component<FlatListProps<number>, {}> {
 }
 
 export class SectionListTest extends React.Component<SectionListProps<string>, {}> {
-    myList: SectionList<any>;
+    myList: React.RefObject<SectionList<any>>;
```

Comment:
Any way we can make this more specific than `any`?

Reply:
Good call